### PR TITLE
Tag Gillespie v0.0.2 [https://github.com/sdwfrost/Gillespie.jl]

### PR DESCRIPTION
**Pull request by sdwfrost**
Trying again with up-to-date repo

### CODE REVIEW

File: Gillespie/versions/0.0.2/requires [hunk 1]
```diff
@@ -0,0 +1,5 @@
+julia 0.4
+Compat
```
Comment (tkelman) [inline Gillespie/versions/0.0.2/requires]: https://github.com/JuliaLang/METADATA.jl/pull/6720#discussion_r82973731

Comment (sdwfrost) [inline Gillespie/versions/0.0.2/requires]: Do you mean a minimum version for Compat as well?

Comment (tkelman) [inline Gillespie/versions/0.0.2/requires]: yes, whatever first implements all the features you need from compat. there's a note on the readme there about how to identify that

**Comment by tkelman** [comment]
it's a good idea to do so if you're relying on any features that haven't been present in every tagged release of those packages, but can be a bit trickier to determine for other packages.

File: Gillespie/versions/0.0.2/requires (hunk 1 above)
Comment (tkelman) [inline Gillespie/versions/0.0.2/requires]: be sure to also make the same change here https://github.com/sdwfrost/Gillespie.jl/blob/master/REQUIRE

**Pull request merged**